src/software/thunderscope/replay/proto_player.py

```diff
@@ -85,22 +109,52 @@ def __sort_replay_chunks(file_path: str):
             replay_index, _ = tail.split(".")
             return int(replay_index)
 
-        self.sorted_chunks = sorted(replay_files, key=__sort_replay_chunks)
+        return sorted(replay_files, key=__sort_replay_chunks)
 
-        # We can get the total runtime of the log from the last entry in the last chunk
-        last_chunk_data = ProtoPlayer.load_replay_chunk(self.sorted_chunks[-1])
+    @staticmethod
+    def is_log_entry_corrupt(log_entry) -> bool:
+        """
+        Check to see if we have can unpack the log entry
+
+        :param log_entry: the log entry we are checking
+        :return: False if we could unpack the log entry, True otherwise
+        """
         try:
-            self.end_time, _, _ = ProtoPlayer.unpack_log_entry(last_chunk_data[-1])
-        except DecodeError:
-            self.end_time, _, _ = ProtoPlayer.unpack_log_entry(last_chunk_data[-2])
-        logging.info(
-            "Loaded log file with total runtime of {:.2f} seconds".format(self.end_time)
-        )
+            _ = ProtoPlayer.unpack_log_entry(log_entry)
+            return False
+        except Exception:
+            return True
 
-        # Start playing thread
-        self.seek(0.0)
-        self.thread = threading.Thread(target=self.__play_protobufs, daemon=True)
-        self.thread.start()
+    def is_it_playing(self) -> bool: 
+        """
+        return whether or not the proto player is being played?
+
+        :return: whether or not proto player is playing!
+        """
```

Comment:
nit:
```suggestion
    def is_playing(self) -> bool: 
        """
        return whether or not the proto player is being played.

        :return: True if the proto player is playing, False otherwise.
        """
```